src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/commands.py

```diff
@@ -151,3 +155,4 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
                   table_transformer=build_output_format)
         g.command('logs', 'acr_build_task_logs', client_factory=cf_acr_builds,
                   table_transformer=None)
+        g.command('cancel', 'acr_build_task_cancel', client_factory=cf_acr_builds)
```

Comment:
nit: wasn't sure if you want to call this as `cancel-build` (and align with `update-build` or `list-builds`) otherwise the cancel seems to be on the build task.

Reply:
Kept the name `cancel` while removed the `client-factory` here.